lib/src/default_extension_map.dart

```diff
@@ -413,6 +413,7 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   'm3a': 'audio/mpeg',
   'm3u': 'audio/x-mpegurl',
   'm3u8': 'application/vnd.apple.mpegurl',
+  'm4a': 'audio/mp4',
```

Comment:
```suggestion
  // Source: https://www.rfc-editor.org/rfc/rfc4337#page-3
  'm4a': 'audio/mp4',
```

@natebosch @kevmoo Should we add the comment like this?